lib/obp/access/elements/table_wrap.rb

```diff
@@ -0,0 +1,37 @@
+module Obp
+  module Access
+    class Renderer
+      class Elements
+        class TableWrap < Base
```

Comment:
Here's an example of `table-wrap` (are inside `<fig>` element):
![image](https://github.com/user-attachments/assets/52e2f883-8dc8-428c-a53e-fd26d93ad1a4)